src/service/repo/signedMessageRepo.ts

```diff
@@ -137,14 +161,29 @@ async function createSignedChat(members: string[]) {
   }
 }
 
+async function deleteMessage(messageId: string) {
+  try {
+    const response = await api.delete(baseUrl.deleteMessage(messageId));
+    if (response.status === 200) {
+      return Promise.resolve(response.data);
+    }
+
+    return Promise.reject(response.data?.status);
+  } catch (e) {
+    console.log(e);
+    return Promise.reject(e);
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `deleteMessage` function sends a DELETE request to the server to delete a specific message. It correctly checks the response status and resolves or rejects the promise accordingly. However, logging the error directly to the console isn't a good practice. Instead, consider throwing an error that can be caught and handled by the calling function.

```diff
async function deleteMessage(messageId: string) {
  try {
    const response = await api.delete(baseUrl.deleteMessage(messageId));
    if (response.status === 200) {
      return Promise.resolve(response.data);
    }

    return Promise.reject(response.data?.status);
  } catch (e) {
-    console.log(e);
+    throw new Error(e);
    return Promise.reject(e);
  }
}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->